docs/nginx.md

```diff
@@ -57,13 +57,19 @@ Read again [the requirements](recipes.md#requirements).
 
 Ready?
 
-Run the following:
+--
+
+Create the required directories
 
 ```
 mkdir -p auth
 mkdir -p data
+```
+
+Create the main nginx configuration you will use.
+
+```
 
-# This is the main nginx configuration you will use
 cat <<EOF > auth/nginx.conf
```

Comment:
I am not sure if we want to encourage copy-pasta. The goal here to teach the components that need to be addressed, rather than provide an all-encompassing configuration.